src/lib/Scenes/Home/Components/TabBar.tsx

```diff
@@ -38,7 +39,7 @@ const TabLabel: any = styled.Text`
   font-size: 13px;
   letter-spacing: 1.5;
   text-align: center;
-  opacity: ${(props: TabLabelProps) => (props.active ? 1.0 : 0.3)};
+  color: ${(props: TabLabelProps) => (props.active ? "black" : colors["gray-medium"])};
```

Comment:
please user `Colors.Black` and `Colors.GrayMedium` next time - see https://github.com/artsy/emission/blob/master/src/lib/data/colors.ts#L1-L18

Reply:
I did so originally but I got lint errors which I was going to ask you about that. I'll re-do shortly, and screenshot them etc, if that's still happening.